gnd/src/main/java/com/google/android/gnd/persistence/local/room/RoomLocalDataStore.java

```diff
@@ -337,11 +340,12 @@ public Completable updateMutations(ImmutableList<Mutation> mutations) {
   }
 
   @Override
-  public Completable finalizePendingMutations(ImmutableList<Mutation> mutations) {
+  public Completable finalizePendingMutations(@Nullable ImmutableList<Mutation> mutations) {
```

Comment:
We can always use an empty immutable list `ImmutableList.of()` instead of null. Why `@Nullable`?

Reply:
Good idea. I've added a null check inside `LocalMutationSyncWorker::processMutations` and removed the `@Nullable` annotation.